src/main.rs

```diff
@@ -18,16 +20,33 @@ pub fn run(args: args::Args) -> Result<(), String> {
     println!("The work interval is set to {} minutes.", args.work);
     println!("The short break interval is set to {} minutes.", args.short);
     println!("The long break interval is set to {} minutes.", args.long);
+    libnotify::init("lightningfocus").unwrap();
```

Comment:
`.unwrap()` panics if the initialization fails. Consider if you want to let `run` return an error in that case instead. Something like:

```rust
libnotify::init("lightningfocus").map_err(|e| format!("Failed initializing libnotify: {e}"))?;
```

If you want to do this, the same goes for the lines on which you show the notifications.

Reply:
Thanks, that's a good suggestion. Changed it.